pandas/tests/resample/test_datetime_index.py

```diff
@@ -751,6 +751,19 @@ def test_resample_base():
     tm.assert_index_equal(resampled.index, exp_rng)
 
 
+def test_resample_float_base():
+    # GH25161
+    dt = pd.to_datetime(["2018-11-26 16:17:43.51",
+                         "2018-11-26 16:17:44.51",
+                         "2018-11-26 16:17:45.51"])
+    s = Series(np.arange(3), index=dt)
+
+    base = 17 + 43.51 / 60
+    resampled = s.resample("3min", base=base).mean()
+    exp_index = DatetimeIndex(["2018-11-26 16:17:43.51"])
+    tm.assert_index_equal(resampled.index, exp_index)
```

Comment:
FWIW, I would prefer to just check the whole `Series`, not the index.

Reply:
I see. Will change it today